### PR TITLE
1inch: project swaps & some linage updates & cleaning

### PR DESCRIPTION
**Pull request by max-morrow**
currently added:
- oneinch_mapped_contracts
- oneinch_mapped_methods
- oneinch_project_swaps_macro

### CODE REVIEW

**Comment by belen-pruvost** [comment]
Hey @max-morrow - @jeff-dude told me that the model `oneinch_base_project_swaps` was failing on this PR. I took the liberty of pushing a new commit to address the underlying issue: when creating tables in trino, rows need to have explicit names. So I changed the way `call_transfer_addresses` is defined. I suggested the row column names `address` and `success` which may not be accurate, so feel free to modify them.

**Comment by grkhr** [comment]
hey @belen-pruvost thanks a lot for your help! will try with this approach then.

File: macros/models/_project/oneinch/oneinch_project_swaps_macro.sql [hunk 1]
```diff
@@ -0,0 +1,162 @@
+{% macro 
+    oneinch_project_swaps_macro(
+        blockchain
+        , date_from = '2024-01-01'
```
Comment (jeff-dude) [inline macros/models/_project/oneinch/oneinch_project_swaps_macro.sql:4]: from my understanding in conversation, initial merge will only read 2024 data. the data will be further tested, then look to do more history in follow up PR.

Comment (jeff-dude) [reply macros/models/_project/oneinch/oneinch_project_swaps_macro.sql:4]: no action on this comment, just clarifying.

Comment (grkhr) [reply macros/models/_project/oneinch/oneinch_project_swaps_macro.sql:4]: yes!

File: macros/models/_project/oneinch/oneinch_project_swaps_macro.sql [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        {% else %}
+            and minute >= timestamp '{{date_from}}'
+        {% endif %}
+)
+
+, creations as (
+    select
+        address
+        , block_number
+    from {{ source(blockchain, 'creation_traces') }}
```
[the remark this answers is not in the document]
Comment (grkhr) [reply macros/models/_project/oneinch/oneinch_project_swaps_macro.sql:88]: unfortunately, we need all data from this table here. but incremental run seems fine so i think it won’t be a problem

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**